src/ir/comp.rs

```diff
@@ -35,7 +35,7 @@ pub enum MethodKind {
 }
 
 /// A struct representing a C++ method, either static, normal, or virtual.
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
I guess I'll figure out why later, but I'm curious. The point of not doing this is that it's both expensive and not quite accurate, given pointing to different `ItemId` doesn't necessarily mean the two `ItemId`s are unequal.